### ISSUE TITLE
Update ADM and mappings to include new ICDC Sample properties 

### ISSUE THREAD

**Issue by mbrush** [cancerDHC/data-model-harmonization#6]
See the four new properties added by PR here:
https://github.com/CBIIT/icdc-model-tool/pull/49

**Comment by mbrush** [comment]
sample_chronology is also new:

https://github.com/CBIIT/icdc-model-tool/blob/master/model-desc/icdc-model-props.yml#L814
Desc: indicator as to when a sample was acquired relative to therapuetic treatment being administered 
Type:
- Before Treatment
- During Treatment
- After Treatment
- Upon Progression
- Upon Relapse
- Upon Death
- Not Applicable
- Unknown
Req: true

**Comment by mbrush** [comment]
From Mark Jensen via Slack:

> If you look now at https://cbiit.github.io/icdc-model-tool/model-desc/, there is a list of excel files - these highlight the changes in the model content with each commit (that led to changes). Should be fairly self-explanatory, but please ping with any questions

**Comment by bfurner** [comment]
@mbrush can you update this issue with any additional information?  assuming the ADM will not continue to be updated, we should likely at least represent these mappings in the CDM.  Is this something that can be incorporated now or do these require additional review?

**Comment by mbrush** [comment]
I agree that the ADM will not be maintained.  Was a great way to get started, but now that we have a model we can add mappings to new properties of included nodes or properties from entirely new nodes, to the CRDCH spreadsheet mapping columns. Let's take a look at these in our next call and close out this ticket?

Also, this ticket raises the general concern about keeping up to date with changes / additions to source models. We had assigned folks to set up a workflow for this, but it was never prioritized and I think Mara was the main driver here and she is no longer on the project.  We should probably revisit and prioritize this after our round of October presentations.
